format/format.go

```diff
@@ -92,6 +103,21 @@ func AstWithOpts(x interface{}, opts Opts) ([]byte, error) {
 			case n.IsEvery():
 				extraFutureKeywordImports["every"] = struct{}{}
 			}
+
+		case *ast.Import:
+			path := n.Path.Value.(ast.Ref)
+			if ast.FutureRootDocument.Equal(path[0]) && len(path) == 2 && path[1].Equal(ast.StringTerm("keywords")) {
+				useContainsKW = true
+				useIf = true
+			}
+
+			if ast.FutureRootDocument.Equal(path[0]) && n.Name() == "contains" {
+				useContainsKW = true
+			}
+
+			if ast.FutureRootDocument.Equal(path[0]) && n.Name() == "if" {
+				useIf = true
+			}
```

Comment:
These allow us to control how rule heads are pretty printed. If the corresponding future keywords are imported, we'll render the rule heads using them. Importing all future keywords will affect pretty-printing, too.

Reply:
Could imagine factoring these conditionals out into the ast package (or perhaps ast/future?) if we find ourselves repeating this logic.